core/src/main/scala/tech/beshu/ror/utils/RCUtils.java

```diff
@@ -56,10 +56,18 @@
       "indices:admin/resolve/index"
   ));
 
+  private static MatcherWithWildcards rorAdminRequestMatcher = new MatcherWithWildcards(Sets.newHashSet(
```

Comment:
let's remove it and do it a little bit differently (in next comments I'll described what I mean)